src/Entity/StandupMeetingReminder.php

```diff
@@ -91,6 +97,24 @@ public function setChannelId(string $channelId): self
         return $this;
     }
 
+    /**
+     * @return array<array-key, int>
+     */
+    public function getForecastClients(): ?array
```

Comment:
```suggestion
    public function getForecastClients(): array
```
can never be null, right?